src/Controls/tests/TestCases.HostApp/Issues/Issue22972.cs

```diff
@@ -1,20 +0,0 @@
-using Microsoft.Maui.Controls;
-
-namespace Maui.Controls.Sample.Issues
-{
-	[Issue(IssueTracker.Github, 22972, "Win platform WebView cannot be release after its parent window get close")]
-	public class Issue22972 : NavigationPage
-	{
-		public Issue22972() 
-		{
-            this.RunMemoryTest(() =>
```

Comment:
I think leaving this test is fine

It's testing a somewhat slightly different scenario